server/src/main.ts

```diff
@@ -4,14 +4,26 @@ import { useContainer } from "class-validator";
 import { AppModule } from "./app.module";
 import { ConfigService } from "./config/config.service";
 import { AppLogger } from "./logger/app-logger";
+import cookieParser from "cookie-parser";
+import { createSession } from "./jira/session-setup";
 
-const options = process.env.NODE_ENV === "development" ? { cors: { origin: "*" } } : {};
+const options =
+  process.env.NODE_ENV === "development"
+    ? {
+        cors: {
+          origin: `http://localhost:${process.env.DEV_FRONTEND_PORT || 3000}`,
```

Comment:
Could you refactor this env var to be called `CORS_ORIGIN`? It would include the whole URL `http://localhost:4000` and subsequently the ternary condition would be just `process.env.CORS_ORIGIN`.

I think this would be make very explicit what this env var is used for and avoid possible, however unlikely, mix-ups in the future.

Also, this variable needs to be added to `docs/configuration.md`.